tools/epanet-output/test/test_epanet_output.cpp

```diff
@@ -0,0 +1,251 @@
+/*
+ *   test_epanet_output.cpp
+ *
+ *   Created: 8/4/2017
+ *   Author: Michael E. Tryby
+ *           US EPA - ORD/NRMRL
+ *
+ *   Unit testing for EPANET Output API using google test.
+*/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string>
+#include "gtest/gtest.h"
+
+#include "../src/epanet_output.h"
+
+#define PROJECT_HOME "C:/Users/mtryby/Workspace/GitRepo/michaeltryby/epanet/"
```

Comment:
should remove hard-coded local paths specific to a machine

Reply:
I understand this is not ideal. The test needs to know the location of the binary file to run. Is there a better way to accomplish this?

Reply:
as a command-line argument passed in by Travis?